bagel/cli.py

```diff
@@ -77,11 +77,11 @@ def pheno(
 
         subject = models.Subject(hasLabel=str(participant))
         if "sex" in column_mapping.keys():
-            subject.hasSex = models.Sex(
-                identifier=putil.get_transformed_values(
-                    column_mapping["sex"], _sub_pheno, data_dictionary
-                ),
+            _sex_val = putil.get_transformed_values(
```

Comment:
Just to note we discussed why this function is currently only returning one value: future support for multiple transformed values.

In this case, that it returns _something_ is the determining factor here.